### ISSUE TITLE
Yelp Ratings Urls seems to have poorly edited photos

### ISSUE THREAD

**Issue by SchlegelThomas** [Yelp/yelp-api#133]
Hi, upon using your api to search for businesses within my application I noticed that some of the ratings images have a non-transparent background.  Normally this wouldn't be a problem if you display them on a white background; however, our site has a dark background and it becomes increasingly noticeable.  I see that you also have some ratings image resources available to download, but those are using an image map and even on the api demo site you are using relative background positioning of the map to display an individual row.  I'd like to avoid that over-engineering if possible in my situation and am wondering if maybe you all can just update your cdn image links? Here is an example: 
![yelpstarfunny](https://cloud.githubusercontent.com/assets/1458527/15364351/e4300d3e-1ce1-11e6-88d6-8c06d8f9a275.PNG)

Let me know if there is an alternate solution as well.  

Thanks,
Thomas

**Comment by mittonk** [comment]
@tomelm , want to have a quick look?

**Comment by tomelm** [comment]
Hey @SchlegelThomas! We're looking into this.

**Comment by SchlegelThomas** [comment]
thanks! let me know when you have an update.

**Comment by SchlegelThomas** [comment]
Any Updates?

**Comment by watterso** [comment]
@SchlegelThomas , this is still on are todo list, but not near the top. In my own personal project I got around this by downloading the image map from [the display requirements page](https://www.yelp.com/developers/display_requirements), chopped it up manually saving them as 1.png, 1.5.png, etc so that I could serve them directly based on the rating field of a business. Of course using the image map itself is probably a more elegant solution, but I liked the simplicity of cutting it up into separate images.

**Comment by SchlegelThomas** [comment]
@watterso Would you mind if I created my own local resource based off of what you've already got then?  I can submit it here for review if you'd like.  I just wanted to make sure we weren't breaking any terms and conditions as far as usage goes.

**Comment by watterso** [comment]
Hey @SchlegelThomas,
After some internal discussion, we've decided that if you want those new stars you should download them from the [display requirements page](https://www.yelp.com/developers/display_requirements). As mentioned on that page, you are not allowed to alter them. The kind of alterations that are forbidden are those that change the physical look of the image. So you can cut out the rows of stars from the image map and serve them how you want, but you can't make them green, sparkly, more red or more yellow, or anything of that sort.

**TL;DR**: use the stars from the [display requirements page](https://www.yelp.com/developers/display_requirements), but don't change how they look.

Closing this out now :)
